### PR TITLE
add method SendTaskCardMessage

### PR DESCRIPTION
**Pull request by zhaikangqi**
添加了发送任务卡片类型的消息

### CODE REVIEW

File: models.go [hunk 1]
```diff
@@ -938,3 +938,12 @@ type respOAGetApprovalDetail struct {
...
 	Info OAApprovalDetail `json:"info"`
 }
+
+// TaskCardBtn 任务卡片消息按钮
+type TaskCardBtn struct {
+	Key         string `json:"key"`
+	Name        string `json:"name"`
+	ReplaceName string `json:"replace_name"`
+	Color       string `json:"color"`
+	IsBold      bool   `json:"is_bold"`
```
Comment (xen0n) [inline models.go:948]: Add field descriptions from official docs.

File: message.go [hunk 2]
```diff
@@ -187,6 +188,38 @@ func (c *WorkwxApp) SendMarkdownMessage(
...
+) error {
+	btnByte, err := json.Marshal(btn)
+	if err != nil {
+		return errors.New("btn convert to json fail: " + err.Error())
+	}
+	var btnSlice []map[string]interface{}
+	err = json.Unmarshal(btnByte, &btnSlice)
+	if err != nil {
+		return errors.New("btn convert to []map fail: " + err.Error())
+	}
```
Comment (xen0n) [inline message.go:209]: This shouldn't be necessary, a `[]TaskCardBtn` is an `interface{}` after all, there's no custom marshaling logic at play.

**Review by xen0n — COMMENTED** [review]
Also update the README about the addition.

File: message.go (hunk 2 above)
Comment (zhaikangqi) [reply message.go]: 是的，我已经删除了这部分代码

**Review by xen0n — APPROVED** [review]
Assuming you tested the code yourself, it should be ready to go. I'd like to rename some fields and make the "button color" string an enum instead, but that can come later.

Thanks for your contribution!

bors r+

**Pull request merged**